pkg/policy/l4.go

```diff
@@ -1663,6 +1676,24 @@ func (l4Policy *L4Policy) AccumulateMapChanges(l4 *L4Filter, cs CachedSelector,
 		}
 		epPolicy.policyMapChanges.AccumulateMapChanges(cs, adds, deletes, keysToAdd, value)
 	}
+	l4Policy.mutex.RUnlock()
+}
+
+// SyncMapChanges marks earlier updates as completed
+func (l4Policy *L4Policy) SyncMapChanges(l4 *L4Filter, handleFunc GetVersionFunc) {
+	// SelectorCache may not be called into while holding this lock!
+	l4Policy.mutex.RLock()
+	for epPolicy := range l4Policy.users {
+		// Skip if endpoint has no policy maps
+		if !epPolicy.PolicyOwner.HasBPFPolicyMap() {
+			continue
+		}
+		if option.Config.Debug {
+			log.Debug("SyncMapChanges")
```

Comment:
Is this a useful debug log?

Reply:
It was when I was debugging this, no longer needed, so removed :-)